active_plugins/measureRWCperObject.py

```diff
@@ -0,0 +1,524 @@
+#################################
+#
+# Imports from useful Python libraries
+#
+#################################
+
+import centrosome.cpmorphology
+import centrosome.zernike
+import numpy
+import scipy.ndimage
+
+#################################
+#
+# Imports from CellProfiler
+#
+##################################
+
+
+__doc__ = """\
+MeasureRWCperObject
+===================
+
+
+"""
+
+#
+# Constants
+#
+# It's good programming practice to replace things like strings with
+# constants if they will appear more than once in your program. That way,
+# if someone wants to change the text, that text will change everywhere.
+# Also, you can't misspell it by accident.
+#
+from cellprofiler_core.constants.measurement import COLTYPE_FLOAT
+from cellprofiler_core.module import Module
+from cellprofiler_core.setting.subscriber import ImageSubscriber, LabelSubscriber
+from cellprofiler_core.setting.text import Integer
+
+"""This is the measurement template category"""
+C_MEASUREMENT_TEMPLATE = "MT"
+M_PER_OBJECT = "Within each object individually"
+
+"""Feature name format for the RWC Coefficient measurement"""
+F_RWC_FORMAT = "Correlation_RWC_%s_%s"
```

Comment:
This may cause a collision if the person has also measured RWC regularly, can we name it something different?